homeassistant/components/bring/__init__.py

```diff
@@ -31,11 +31,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     password = entry.data[CONF_PASSWORD]
 
     session = async_get_clientsession(hass)
-    bring = Bring(email, password, sessionAsync=session)
+    bring = Bring(session, email, password)
 
     try:
-        await bring.loginAsync()
-        await bring.loadListsAsync()
+        await bring.login()
+        await bring.loadLists()
```

Comment:
I recommend following PEP8 and use snake_case method names.

Reply:
Let me first switch the lib and then improve upon. You usually don't want multiple changes in the same PR. It is planned